services/continuous_querier/service.go

```diff
@@ -274,7 +271,7 @@ func (s *Service) runContinuousQueries(req *RunRequest) {
 // ExecuteContinuousQuery may execute a single CQ. This will return false if there were no errors and the CQ was not run.
 func (s *Service) ExecuteContinuousQuery(dbi *meta.DatabaseInfo, cqi *meta.ContinuousQueryInfo, now time.Time) (bool, error) {
 	// TODO: re-enable stats
-	//s.stats.Inc("continuousQueryExecuted")
+	// s.stats.Inc("continuousQueryExecuted")
```

Comment:
remove this?

Reply:
not even sure where this came from.  will remove it.